LICENSE

```diff
@@ -3,6 +3,9 @@ BSD 2-Clause License
 Copyright (c) 2022, Jake Barter
 All rights reserved.
 
+Copyright (c) 2020, MasterKenth
+All rights reserved.
```

Comment:
Thoughts on including reason why something is included in the license?
```suggestion
Copyright (c) 2020, MasterKenth
All rights reserved.
Drop rate information from the Discord Rare Drop Notifier plugin
```

Reply:
I'm actually in the process of freshly generating the json from wiki parsing via python scripts, so we'll have the latest drop rates (& not need to update our license)

Reply:
Nice! Can be used with a github actions cronjob to keep us up to date too

Reply:
Updated in https://github.com/pajlads/DinkPlugin/pull/141/commits/88331ec8f35cface89444d6c4575a0e5127c439e - will do scripts+cronjob in another pr